modules/fbx/doc_classes/EditorSceneImporterFBX.xml

```diff
@@ -7,24 +7,15 @@
 		This is an FBX 3D asset importer with full support for most FBX features.
 		If exporting a FBX scene from Autodesk Maya, use these FBX export settings:
 		[codeblock]
-		- Smoothing Groups
-		- Smooth Mesh
+		- Leave settings at default in most exporters. (settings below in Maya are mostly irrelevant as they are the defaults.)
```

Comment:
```suggestion
		- Leave default settings in most exporters. (Settings below are mostly irrelevant in Maya as they are the defaults.)
```
Which brings the question though, why bother mentioning things if they're not relevant? :)

Reply:
This was before my time, like those settings are so old, I was surprised they were still part of the engine.